schema/derive/Cargo.toml

```
@@ -9,6 +9,11 @@ publish = false
 proc-macro = true
 
 [dependencies]
-proc-macro2 = "1.0"
+syn = { version = "1.0.72", default-features = false, features = ["full"] }
 quote = "1.0"
-syn = "1.0.72"
+proc-macro2 = "1.0"
```

With regards to your comment on the new `CONTRIBUTING.md` should we use 
```suggestion
proc-macro2 = "1.0.32"
```